src/include/common/concurrent_vector.h

```diff
@@ -125,6 +125,18 @@ class ConcurrentVector {
    */
```

Comment:
This class should stay in common/container since it's a container.

Reply:
done